internal/cloudsql/instance.go

```diff
@@ -171,7 +171,10 @@ type ConnectionInfo struct {
 	ConnectionName    instance.ConnName
 	ClientCertificate tls.Certificate
 	ServerCaCert      *x509.Certificate
+	ServerCaCertPem   []byte // Only set for CAS instances.
```

Comment:
Let's make this ServerCACertPEM (and the above should be ServerCACert). Ditto below wherever we use CA (i.e., "CA" not "Ca", "PEM", not "Pem").

Reply:
We probably want have a single field here `ServerCaCert []*x509.Certificate`, instead of two fields populated differently depending on ServerCaMode

Reply:
Agreed -- if we can do this with one field, that's best.

Reply:
Thanks. Used the `ServerCACert []*x509.Certificate` only now.